code/modules/admin/verbs/striketeam.dm

```diff
@@ -35,16 +35,16 @@ var/const/commandos_possible = 6 //if more Commandos are needed in the future
 		to_chat(usr, "<font color='red'>Someone is already sending a team.</font>")
 		return
 
-	if(alert("Do you want to send in a strike team? Once enabled, this is irreversible.",,"Yes","No")!="Yes")
+	if(tgui_alert(src, "Do you want to send in a strike team? Once enabled, this is irreversible.", "Confirmation", list("Yes","No"))!="Yes")
 		return
 
-	alert("This 'mode' will go on until everyone is dead or the installation is destroyed. You may also admin-call the evac shuttle when appropriate. Spawned commandos have internals cameras which are viewable through a monitor inside the Spec. Ops. Office. Assigning the team's detailed task is recommended from there. While you will be able to manually pick the candidates from active ghosts, their assignment in the squad will be random.")
+	tgui_alert(src, "This 'mode' will go on until everyone is dead or the installation is destroyed. You may also admin-call the evac shuttle when appropriate. Spawned commandos have internals cameras which are viewable through a monitor inside the Spec. Ops. Office. Assigning the team's detailed task is recommended from there. While you will be able to manually pick the candidates from active ghosts, their assignment in the squad will be random.")
 
 	choice = null
 	while(!choice)
 		choice = sanitize(input(src, "Please specify which mission the strike team shall undertake.", "Specify Mission", ""))
 		if(!choice)
-			if(alert("Error, no mission set. Do you want to exit the setup process?",,"Yes","No")=="Yes")
```

Comment:
double commas here, i may have missed others do a sweep of the whole branch for doubled commas

Reply:
Double commas mean there will be no window title